HUD/EndTextCommandIsThisHelpMessageBeingDisplayed.md

```diff
@@ -6,11 +6,12 @@ aliases: ["0x10BDDBFC529428DD"]
 
 ```c
 // 0x10BDDBFC529428DD 0xF63A13EC
-BOOL END_TEXT_COMMAND_IS_THIS_HELP_MESSAGE_BEING_DISPLAYED(int p0);
+BOOL END_TEXT_COMMAND_IS_THIS_HELP_MESSAGE_BEING_DISPLAYED(int hudIndex);
 ```
 
 
 ## Parameters
-* **p0**: 
+* **hudIndex**: The hud index for help message. `0` is `HELP_TEXT`, `1` and `2` is `FLOATING_HELP_TEXT_1` and `FLOATING_HELP_TEXT_2`.
 
 ## Return value
+```true```: Indicates if the specific help message has been displayed. ```false```: Indicates that the specific help message is still being displayed.
```

Comment:
Hi, why did you choose to use triple back ticks? I think single back ticks would do just as well.

Reply:
Just a misclick, I will change it now.